src/Meditation.Bootstrap.Native/EntryPoint.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Meditation.Interop;
+using Meditation.Interop.Windows;
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+
+namespace Meditation.Bootstrap.Native
+{
+    public static class EntryPoint
+    {
+        // Note: this method is dynamically invoked by tests
+        [UnmanagedCallersOnly(EntryPoint = "MeditationSanityCheck")]
+        public static uint SanityCheck(IntPtr _)
+        {
+            return 0xABCD_EF98;
```

Comment:
I will add comments.

This is a method that is used in tests only. A native module gets loaded into target process and to establish that everything works, injector will attempt to call this method and validate that it will receive the `0xABCD_EF98` constant.

Therefore, this is not an address. ASLR has no effect on this. The only assumption that I make is that `kernel32` is loaded at the same address for processes. But I think this holds and is used by a lot of similar software.